change_log/next/FE-1974-date-onchange.yml

```diff
@@ -0,0 +1,7 @@
+New Features: "(Component: DateInput) wrapped with the `withUniqueName`/HOC. Added `name` property to action/onChange event."
```

Comment:
Please could you move the all (Component: DateInput) text to the end of the line to be consistent with the other yaml files in this release.

Reply:
updated